skaffold/manifests/config.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: vector-config
+data:
+  vector.toml: |
```

Comment:
Look it's the dreaded toml in yaml

Reply:
Yeah, meet the kubernetes configmaps.

Actually, since we're using `kustomize`, we can unwrap the inner config and pass it through the kustomize `configMapGenerator` instead of `resources` clause.

I used `kubectl` `create/delete` flow initially, and missed this part when transitioned to `kubectl kustomize`.

At the same time - maybe it's better to keep like this for a while, until the turbulence with `skaffold`'s `0.10` release stabilizes.

Reply:
I think it's fine, was more just amusing.